product_listings_manager/tests/test_rest_api_v1.py

```diff
@@ -4,22 +4,25 @@
 
 from product_listings_manager.app import app
 
-# Flask < 1.0 does not have get_json.
-from flask import Response
-import json
-if not hasattr(Response, 'get_json'):
-    def get_json(self, force=False, silent=False, cache=True):
-        """ Minimal implementation we can use this in the tests. """
-        return json.loads(self.data)
-    Response.get_json = get_json
-
 
 @pytest.yield_fixture
 def client():
     client = app.test_client()
     yield client
 
 
+class TestIndex(object):
+    def test_get_index(self, client):
+        r = client.get('/api/v1.0/')
+        expected_json = {
+            'about_url': 'http://localhost/api/v1.0/about',
+            'product_info_url': 'http://localhost/api/v1.0/product-info/:label',    # NOQA: E501
+            'product_listings_url': 'http://localhost/api/v1.0/product-listings/:label/:build_info'  # NOQA: E501
```

Comment:
You can remove these NOQAs, since #32 was merged.

Reply:
Good call, done.